frontend/src/components/common/comments-list/helpers/check-is-liked/check-is-liked.ts

```diff
@@ -0,0 +1,10 @@
+import { Comment } from 'common/types/types';
```

Comment:
`frontend/src/components/common/comments-list/helpers/check-is-liked/check-is-liked.helper.ts`